pyfive/dataobjects.py

```diff
@@ -226,18 +228,28 @@ def _parse_attribute_msg(self, buffer, offset):
                     f"Attribute {name} type not implemented, set to None."
                 )
             return name, None
+
         offset += _padded_size(attr_dict['datatype_size'], padding_multiple)
 
         # Read the dataspace information
         shape, maxshape = determine_data_shape(buffer, offset)
-        items = int(np.prod(shape))
+        items = prod(shape)
```

Comment:
I guess this depends on shape being an integer, so you can remove the redundancy?